resources/seccomp/aarch64-unknown-linux-musl.json

```diff
@@ -459,6 +459,19 @@
                     }
                 ]
             },
+            {
+                "syscall": "fcntl",
+                "comment": "Used by MMDS",
```

Comment:
Maybe write a more verbose comment here, explain why MMDS needs it

Reply:
Update: since we now allow both MMDS versions, `fcntl` is required when setting V2 after Firecracker has started. I added comments to mention why MMDS needs to have this syscall on the allow list.